features/storage/TESTS/blockdevice/general_block_device/main.cpp

```diff
@@ -209,6 +213,140 @@ void test_multi_threads()
     TEST_ASSERT_EQUAL(0, err);
 }
 
+void test_get_erase_value()
+{
+    utest_printf("\nTest BlockDevice::get_erase_value()..\n");
+
+    BlockDevice *block_device = BlockDevice::get_default_instance();
+    TEST_ASSERT_NOT_NULL(block_device);
```

Comment:
the TEST_ASSERT_NOT_NULL will fail all platforms that receive NULL from get_default_instance, notice that are more than a few in our CI